### PR TITLE
resolves #13180, changed "Foren" to "Gruppen" in german translation

### LINKED ISSUE #13180: Normalize @-tag/@-mention in hosted documentation
Both "@-tag" and "@-mention" are used in the hosted documentation, we need to pick one and replace all the other occurrences.

### PR DESCRIPTION
**Pull request by Raroun**
- resolves #13180 
- changed forums to groups in german translation

### CODE REVIEW

File: doc/de/Forums.md
Comment (MrPetovan) [inline doc/de/Forums.md:1]: Please restore this file and replace the `@-Tag` reference.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: doc/de/Forums.md
Comment (Raroun) [reply doc/de/Forums.md:1]: Done.

File: doc/de/Home.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Friendica - Dokumentation und Ressourcen
 	* [Freunde finden](help/Making-Friends)
 	* [Circles und Privatsphäre](help/Circles-and-Privacy)
 	* [Tags und Erwähnungen](help/Tags-and-Mentions)
-	* [Community-Foren](help/Forums)
+	* [Community-Gruppen](help/Groups)
```
Comment (MrPetovan) [inline doc/de/Home.md]: Part of this change won't work if the corresponding file isn't renamed.
```suggestion
	* [Community-Gruppen](help/Forums)
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Thank you for your work!

**Comment by Raroun** [comment]
Thank you for reviewing  :)

**Comment by MrPetovan** [comment]
Best part of the job!

**Pull request merged**